### ISSUE TITLE
macOS Electron build

### ISSUE THREAD

**Issue by todrobbins** [HughIsaacs2/DatPart#6]
I didn't see a macOS Electron build on the to-do list. Any plans for that?

Thanks!

**Comment by HughIsaacs2** [comment]
Yes! (I didn't even think to add this to the to-do list, it just sort of made sense in my head)

Sadly right now, I don't have a working/up-to-date Mac.

I'm literally considering compiling a Mac OS version at a friend's house soon.
